lib/Wikia/src/Service/User/Attributes/AttributeKeyValueService.php

```diff
@@ -47,7 +47,7 @@ public function set( $userId, Attribute $attribute ) {
 
 			return $ret;
 		} catch ( \Exception $e ) {
-			$this->logError( $userId, $e );
+			$this->logError( $userId, $e, "USER_ATTRIBUTE error saving to service" );
```

Comment:
Should they begin with `USER_ATTRIBUTES` (plural) to match with https://kibana.wikia-inc.com/index.html#/dashboard/temp/AVOwK8vewuxBx3joFaRw ?